syncer/syncer.go

```diff
@@ -1360,6 +1381,14 @@ func (s *Syncer) syncDML(
 		case sqlJob, ok := <-jobChan:
 			metrics.QueueSizeGauge.WithLabelValues(s.cfg.Name, queueBucket, s.cfg.SourceID).Set(float64(len(jobChan)))
 			if !ok {
+				tctx.L().Info("close job chan", zap.Any("rest job", jobs))
```

Comment:
I think we can change ctx in L1340-1342 depend on input ctx directly instead of background

Reply:
@lance6716 @lichunzhu PTAL, I don't know why use background.

Reply:
I remember the original background context is used to make sure after user paused the task, some remaining job could still be executed because the `tctx` is passed from `(*SubTask).run`, and will be canceled when `(*SubTask).callCurrCancel`

Now in `(*Syncer).Run` we have changed the context chain, I'll take a look soon